modules/agent/pkg/controllers/bundledeployment/controller.go

```diff
@@ -252,3 +279,72 @@ func readyError(status fleet.BundleDeploymentStatus) error {
 
 	return errors.New(msg)
 }
+
+func hasAutoReapply(bd *fleet.BundleDeployment, status *fleet.BundleDeploymentStatus) (ok bool, requeueAfter time.Duration, err error) {
```

Comment:
Idealy if you make this method only return `bool` it is much more readable. You don't have to return `requeueAfter` as it can be computed as outside of this method.

Reply:
done